### DISCUSSION TITLE
Can't open midi editor

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1183] [Team Posts]
Hey Kybos,

I'm trying to get some testing done with the new layout but just noticed that I seem to be unable to open a midi or drum editor from by clicking on a part.
I'm sure I tried this before (possibly not on this computer) but now nothing happens. Any ideas?

**Comment by kybos** [comment]
I'm doing that all the time, never had a problem with it, weird.
I will check the current master in my KDE VM, I guess KDE is what you're using.

**Comment by kybos** [comment]
Even on KDE it's working as expected.

**Comment by spamatica** [comment]
I found the problem. The "open in tab" checkboxes were not checked and there might be something wrong with opening in a separate window. Opening in tab works fine when checked.

Just noticed that tooltips are now displayed in the window footer, great usability improvement!

**Comment by kybos** [comment]
Right, can reproduce. Will check.

**Comment by kybos** [comment]
Fixed, it should work again.
